client/CServerHandler.cpp

```diff
@@ -262,8 +262,8 @@ void CServerHandler::justConnectToServer(const std::string & addr, const ui16 po
 		}
 		catch(std::runtime_error & error)
 		{
-			logNetwork->warn("\nCannot establish connection. %s Retrying in 1 second", error.what());
-			boost::this_thread::sleep_for(boost::chrono::milliseconds(1000));
+			logNetwork->warn("\nCannot establish connection. %s Retrying in 10 ms", error.what());
```

Comment:
Might lead to spam in logs as result.

Reply:
That as well as what's printed in the CConnection constructor:

```
Establishing connection...
Found endpoints:
        0:127.0.0.1:3030
Trying connection to 127.0.0.1:3030(0)
```
I could print this information only in case the connection fails, would that be alright?